pom.xml

```diff
@@ -2358,12 +2358,6 @@ under the License.
 								<exclude>@org.apache.flink.annotation.PublicEvolving</exclude>
 								<exclude>@org.apache.flink.annotation.Internal</exclude>
 								<!-- MARKER: start exclusions; these will be wiped by tools/releasing/update_japicmp_configuration.sh -->
```

Comment:
If we could drop the exclusions, do we still need the markers here?

Reply:
This  marker is used for location by the script, see: https://github.com/apache/flink/blob/release-1.19/tools/releasing/update_japicmp_configuration.sh#L63